pom.xml

```diff
@@ -333,8 +333,8 @@
         <oltu.version>1.0.0.wso2v3</oltu.version>
         <oltu.package.import.version.range>[1.0.0, 2.0.0)</oltu.package.import.version.range>
         <json.wso2.version>3.0.0.wso2v1</json.wso2.version>
-        <nimbusds.version>7.3.0.wso2v1</nimbusds.version>
-        <nimbusds.osgi.version.range>[7.3.0,8.0.0)</nimbusds.osgi.version.range>
+        <nimbusds.version>9.37.3.wso2v1</nimbusds.version>
+        <nimbusds.osgi.version.range>[9.0.0,11.0.0)</nimbusds.osgi.version.range>
```

Comment:
Any special reason to use 11 instead of 10 as the closing version of the range?

Reply:
No any special reason, followed the same as https://github.com/wso2-extensions/identity-oauth2-grant-token-exchange/pull/24/files,

Reply:
Can you try with 10 as the upper bound and see if it is working?